Sources/FoundationInternationalization/Formatting/ByteCountFormatStyle.swift

```diff
@@ -152,9 +152,9 @@ public struct ByteCountFormatStyle: FormatStyle, Sendable {
             return false
         }
 
-        func _format(_ value: ICUNumberFormatter.Value) -> AttributedString {
```

Comment:
Sorry I spoke too soon. This is actually used internally by Measurement formatting (source code not available yet but will be soon) where we're passing in `value: .floatingPoint()`. Do you think we can find a way to keep this around?

The ideas I have in mind are (1) keep the initial approach you took in this PR, where we'd just make an educated guess with `isZero`, or(2) add an explicit `func _format(value: Double)` and flesh out sharable guts from the format function to share between these two types. 

What do you think? Any other ideas?

Reply:
I feel bad asking for (2) since that's really not part of this PR. I can make a follow up change on that if you want to revert to (1)

Reply:
I see, no worries. I've tried a few solutions, and this one was the least complicated: ba8bf28aba71e70b476ab303203487346f7e1d4c (`+10`, `-6`). It's basically (2), but parameterized as `_format(_:doubleValue:)`. It's a small change.

```swift
_format(.floatingPoint(value), doubleValue: value)
```